flyway/scxa/migrations/V8__scxa-create-collections.sql

```diff
@@ -0,0 +1,42 @@
+create table collections
+(
+	coll_id varchar default 255 not null,
+	name varchar default 255 not null,
+	description varchar default 255
+);
+
+create unique index collections_coll_id_uindex
+	on collections (coll_id);
+
+alter table collections
+  add constraint collections_pk
+  		primary key (coll_id);
+
+comment on table collections is 'SCXA Experiment collections';
+
+create unique index experiment_accession_uindex
+	on experiment (accession);
+
+alter table experiment
+	add constraint experiment_pk
+		primary key (accession);
```

Comment:
Quite an oversight that `experiment` hasn’t got an index!

Reply:
yeah, I know 😂 !!